diff --git a/.env b/.env
index 74b763a..fb07ac1 100644
--- a/.env
+++ b/.env
@@ -5,6 +5,6 @@ DOCKER_REGISTRY=docker.io
 ENROLL_ID=admin
 ENROLL_SECRET=adminpw
 COMPOSE_CONVERT_WINDOWS_PATHS=1
-FABRIC_STARTER_VERSION=baas-test-master
-FABRIC_VERSION=1.4.4
-FABRIC_STARTER_REPOSITORY=alxkilpio
+FABRIC_STARTER_VERSION=latest
+FABRIC_VERSION=1.4.9
+FABRIC_STARTER_REPOSITORY=olegabu
diff --git a/deploy.sh b/deploy.sh
index 8c486ab..26defd0 100755
--- a/deploy.sh
+++ b/deploy.sh
@@ -38,22 +38,6 @@ export FABRIC_STARTER_VERSION=${FABRIC_STARTER_VERSION:-baas-test}
 source ${first_org}_env;
 #export ENROLL_SECRET=`echo ${ENROLL_SECRET/!/\\\\!}`
 
-
-if [ "$DEPLOY_VERSION" == "Hyperledger Fabric 1.4.4-GOST-34" ]; then
-    set -x
-    export DOCKER_REGISTRY=45.12.73.98
-    export FABRIC_VERSION=gost
-    export FABRIC_STARTER_VERSION=gost
-    export AUTH_MODE=ADMIN
-    export CRYPTO_ALGORITHM=GOST
-    export SIGNATURE_HASH_FAMILY=SM3
-
-    sudo mkdir -p /etc/docker/certs.d/${DOCKER_REGISTRY}
-    openssl s_client -showcerts -connect ${DOCKER_REGISTRY}:443 </dev/null 2>/dev/null|openssl x509 -outform PEM \
-        | sudo tee /etc/docker/certs.d/${DOCKER_REGISTRY}/ca.crt
-    set +x
-fi
-
 info "Cleaning up"
 ./clean.sh all
 
